### ISSUE TITLE
Tables in Rich Text Fields

### ISSUE THREAD

**Issue by freinhard** [OneGov/onegov.town#50]
Tables added via the Redactor WYSIWYG Editor. Make sure tables are responsive.

**Comment by href** [comment]
I don't think we should make those responsive at all. The only reason people want tables seems to be the wish to do page layouts. If we scramble those layouts up we will probably make the content somewhat unusable. Responsiveness goes very much against WYSIWYG here.

As for tables.. I'm somewhat okay with them, though I'm convinced that they are just symptoms of a client wanting to do something we don't offer as a proper feature.

**Comment by href** [comment]
-1 on tables because they go against our promise of accessibility. We use tables we end up with sites which are not accessible for people with screen-readers.

Since that excludes tables with actual data, we could offer a way to push numbers via Excel or a special input mask, making it impossible to use it for layouting.

I think on this we should take a strong stance, because once we have tables people can push images the way they like, change layouts completely and make it really hard for disabled people to use their site.

So this Ticket should be used to gather all the actual use-cases where customers demand tables and find solutions for those use-cases without adding a general table widget.

**Comment by href** [comment]
See also http://maurits.vanrees.org/weblog/archive/2015/10/rob-porter-how-you-can-become-an-accessibility-superhero

**Comment by href** [comment]
We have to remember, government is *required* to provide accessible websites. It's not a nice-to-have.
